src/coreclr/inc/corinfo.h

```diff
@@ -2719,6 +2719,13 @@ class ICorStaticInfo
             CORINFO_CLASS_HANDLE       *vcTypeRet       /* OUT */
             ) = 0;
 
+    virtual CorInfoTypeWithMod getArgType2 (
+            CORINFO_SIG_INFO*           sig,            /* IN */
+            CORINFO_ARG_LIST_HANDLE     args,           /* IN */
+            CORINFO_CLASS_HANDLE       *vcTypeRet,      /* OUT */
+            int *flags = NULL
```

Comment:
This still needs a comment on what these `int flags` mean.

Reply:
OK, I will do it tomorrow morning.

Reply:
Hi, @jkotas 
I had finished the comments for it.
Should I rename the function `getFieldTypeByHnd` within this PR ?
liking https://github.com/dotnet/runtime/pull/62885#discussion_r783496388

Reply:
Yes, that name sounds better.

Reply:
OK, I will update it.